src/coreclr/nativeaot/Runtime.Base/src/System/Runtime/InternalCalls.cs

```diff
@@ -62,12 +62,12 @@ internal static class InternalCalls
         [RuntimeExport("RhCollect")]
         internal static void RhCollect(int generation, InternalGCCollectionMode mode, bool lowMemoryP = false)
         {
-            RhpCollect(generation, mode, lowMemoryP);
+            RhpCollect(generation, mode, lowMemoryP ? Interop.BOOL.TRUE : Interop.BOOL.FALSE);
         }
 
         [DllImport(Redhawk.BaseName)]
         [UnmanagedCallConv(CallConvs = new Type[] { typeof(CallConvCdecl) })]
-        private static extern void RhpCollect(int generation, InternalGCCollectionMode mode, bool lowMemoryP);
+        private static extern void RhpCollect(int generation, InternalGCCollectionMode mode, Interop.BOOL lowMemoryP);
```

Comment:
The default marshalling behavior in CoreCLR is to marshal that as an `int`, but I don't know about NAOT. Regardless of that, this will make the signature blittable which means inlinable and easier to work with.

Thanks!